tests/pytests/test_cassini_drivers.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+import ale
+from ale.drivers import cassini_drivers
+from ale.base import data_naif
+from ale.base import label_pds3
+from ale.formatters import usgscsm_formatter
+
+from unittest.mock import PropertyMock, patch
+
+# 'Mock' the spice module where it is imported
+from conftest import SimpleSpice, get_mockkernels
+
+simplespice = SimpleSpice()
+
+data_naif.spice = simplespice
+cassini_drivers.spice = simplespice
+label_pds3.spice = simplespice
+
+from ale.drivers.cassini_drivers import CassiniIssPds3LabelNaifSpiceDriver
+
+CassiniIssPds3LabelNaifSpiceDriver.metakernel = get_mockkernels
+
+c = CassiniIssPds3LabelNaifSpiceDriver("")
```

Comment:
Pass this as a fixture to each test. Don't use a global variable.

Reply:
I switched it to a fixture. I'm not too familiar with them so let me know if it looks alright.